File: GraduateStudyWebApp/ExamLibSpa_UT/Pages/TS/PubFun/clsCboObject.ts
﻿/*
 * 功能：为Web提供下拉框对象类
 * 版本：2019-08-03-01
 * 作者：潘以锋
 * */

/// <summary>
/// 下拉框对象类
/// </summary>
export class clsCboObject {

    /*
    /// <param name="strValue"></param>
    /// <param name="strText"></param>
    */
    constructor(strValue: string, strText: string) {
        this.Value = strValue;
        this.Text = strText;
    }
    /// <summary>
    /// 值
    /// </summary>
    public Value: string = "";
    /// <summary>
    /// 文本
    /// </summary>
    public Text: string = "";
    /// <summary>
    /// 常量:"Text"
    /// </summary>
    public static get con_Text(): string { return "Text"; };    //文本
    /// <summary>
    /// 常量:"Value"
    /// </summary>
    public static get con_Value(): string { return "Value"; }    //值

}

File: TzPubFunction/com/taishsoft/util/clsCboObject.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace com.taishsoft.util
{
    /// <summary>
    /// 下拉框对象类
    /// </summary>
    public class clsCboObject
    {
        /// <summary>
        /// 
        /// </summary>
        public clsCboObject()
        {
        }
        /// <summary>
        /// 
        /// </summary>
        /// <param name="strValue"></param>
        /// <param name="strText"></param>
        public clsCboObject(string strValue, string strText)
        {
            mstrValue = strValue;
            mstrText = strText;
        }
        private string mstrValue;
        /// <summary>
        /// 值
        /// </summary>
        public string Value
        {
            get { return mstrValue; }
            set { mstrValue = value; }
        }
        private string mstrText;
        /// <summary>
        /// 文本
        /// </summary>
        public string Text
        {
            get { return mstrText; }
            set { mstrText = value; }
        }
        /// <summary>
        /// 常量:"Text"
        /// </summary>
        public const string con_Text = "Text";    //文本

        /// <summary>
        /// 常量:"Value"
        /// </summary>
        public const string con_Value = "Value";    //值
    }
}
